### PR TITLE
Update meshviewer links for FF communities

### PR DESCRIPTION
**Pull request by maurerle**
update link für königswinter, bielefeld and aachen
lüneburg are part of bielefeld now

### CODE REVIEW

File: communities.json [hunk 1]
```diff
@@ -115,7 +115,8 @@
     "https://falterturm.freifunk-vec.de/meshviewer.json"
   ],
   "Königswinter": [
-    "https://ffkwkarte.freifunk-koenigswinter.de/data/koenigswinter/meshviewer.json"
+    "https://ffkwsn03.freifunk-koenigswinter.de/data/meshviewer.json",
+    "https://map.divonet.de/data/sn04-netsplit/meshviewer.json
```
Comment (AiyionPrime) [inline communities.json:119]: This on lacks a trailing `"`

**Review by AiyionPrime — CHANGES_REQUESTED** [review]

File: communities.json (hunk 1 above)
Comment (maurerle) [reply communities.json]: Sorry, this is fixed now.

File: communities.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "https://karte.freifunk-ansbach.de/data/nodes.json"
   ],
   "Bielefeld": [
-    "https://map.freifunk-bielefeld.de/meshviewer/data/meshviewer.json"
+    "https://map03.4830.org/data/nodes.json"
```
Comment (AiyionPrime) [inline communities.json:12]: This nodes.json reveals 566 nodes, while Bielefeld has 87?

**Review by AiyionPrime — COMMENTED** [review]
Lüneburgs note is here:
https://freifunk-lueneburg.de/netzumstellung/

Bielefeld does possibly have less nodes than whats reported as the current url.

File: communities.json (hunk 2 above)
Comment (herbetom) [reply communities.json:12]: the json is also already included as gütersloh

https://github.com/freifunk-gluon/census-exporter/blob/895e2fd4c7b9d1afee092464f31f92cf3331cf93/communities.json#L81-L83

Comment (maurerle) [reply communities.json:12]: Oh well, then it is probably better to separate the communities correctly as given here:
https://map03.4830.org/

@herbetom weird that I missed that..

Comment (herbetom) [reply communities.json:12]: I don't quite see the point in seperating them. From my understanding it's not like their using different Firmwares.

Comment (maurerle) [reply communities.json]: @herbetom yes thats true, but separating them would improve just the naming.
Having all those communities under the name "Gütersloh" seems weird too.

What would you suggest instead?
Renaming to Gütersloh/Bielefeld or 4830.org ?

Comment (herbetom) [reply communities.json]: Splitting would mean having to deal with a many-fold number of timeouts in the case of problems.
In addition i don't think it's relevant for the census-exporter where exactly Nodes are located. What might matter is how you can find a map/website for the community that includes them.

So yeah, maybe 4830.org or leave it at Gütersloh. I don't think Gütersloh/Bielefeld would be the way to go.

**Comment by maurerle** [comment]
Thanks. I think there are no issues open anymore.

**Review by AiyionPrime — APPROVED** [review]
Nur aus Interesse,
was ist die Trennung in Chemnitz zwischen Brewster und Descartes?

**Comment by maurerle** [comment]
Brewster ist wohl Domain "Chemnitz" während Descartes alle anderen Domains (Umland und Co) enthält.
Sind wohl verschiedene Monitoring-Server die ihren Fuß in verschiedenen Teilen des Netzes drin haben.

Könnte man sicher auch mit Yanic "database.connection.respondd" weiterleitung lösen, aber passt ja so auch.

**Comment by maurerle** [comment]
What is blocking this currently?

**Comment by AiyionPrime** [comment]
@herbetom reviewst und mergst du?

File: communities.json [hunk 3]
```diff
@@ -115,7 +113,8 @@
     "https://falterturm.freifunk-vec.de/meshviewer.json"
   ],
   "Königswinter": [
```
Comment (herbetom) [inline communities.json:115]: Maybe Königswinter should be renamed to DiVoNet. From the looks of it that seems to be a construct that is somewhat regional but if you zoom the card out you get quite a few nodes (freifunk-thw.de) outside of that area ...

But idk, can also stay like this for now.

**Review by herbetom — APPROVED** [review]
@AiyionPrime I've got no permissions to merge this. I could hoewer deploy it once it's merged.

File: communities.json (hunk 3 above)
Comment (maurerle) [reply communities.json]: As the old map at https://ffkwsn03.freifunk-koenigswinter.de/data/meshviewer.json did not contain any nodes besides the single supernode, I renamed the community to Divonet in my latest Commit.
Thanks!

**Comment by AiyionPrime** [comment]
Ah sorry. @mweinelt do you want to merge this?

**Comment by mweinelt** [comment]
and deployed.

**Pull request merged**